Sources/KlaviyoSDKDependencies/ComposableArchitecture/Effect.swift

```diff
@@ -0,0 +1,371 @@
+@preconcurrency import Combine
+import KlaviyoSDKDependencies
```

Comment:
```suggestion
```
we shouldn't need this because it's already part of the `KlaviyoSDKDependencies` package